scss/index.scss

```diff
@@ -38,10 +38,39 @@ body {
 }
 
 p {
-  font-size: 15px;
+  font-size: 16px;
```

Comment:
How does this impact other pages? Do we really want this across the app? This change isn't scoped.

Reply:
The better question is why it was needed in the first place? If you compare notion and self-hosted lesson you can notice that the latter looks a bit better. I suppose notion spent quite some time optimizing for the best reading experience, so I tried to copy it as much as I can. `16px` is the default font size anyways.

Reply:
Shouldn't we be using `em` units instead of absolute `px` units anyways to keep size scaling pixel perfect?

Reply:
`px` are not really pixels, they are some sort of pseudo units which are tied to physical screen size, so it's not that bad. For example, hero image on the landing page uses hardcoded width and height in pixels (just like most next/images) and it will look good every screen from small old phones to modern 4k monitors. 

I wouldn't use `em`s for fonts because they depend on parent component, so `em` in on place won't be the same as `em` somewhere else. 

`rem` is tied to root font size and it's the safest way to define font-size. It will scale if user zooms the page and it will be the same everywhere. 

At least that how I understand it, don't quote me on that.